dns-resolver/README.md

```diff
@@ -0,0 +1,20 @@
+# dns-resolver
+==============
+
+This module is a replacement of the [dns addon](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns). The principal difference is the use of [unbound](https://www.unbound.net) as caching DNS resolver that forwards queries for the kubernetes domain to skydns and uses an external DNS server for other queries (like in skydns). One interesting feature of unbound is the extraction of [statistics](https://www.unbound.net/documentation/howto_statistics.html). The image built with this module uses [muning](https://github.com/munin-monitoring/munin) and a web server to access this stats in a graphical way.
```

Comment:
> How is it a replacement for skydns if it forwards to skydns?

In this PR I forward the queries for the local domain
(I'm adding a second commit that replaces skydns and etcd containers)